rfcs/rfc-involuntary-dissolutions.md

```diff
@@ -0,0 +1,210 @@
+
+- Start Date: 2024-04-26
+- Target Major Version:
+- Reference Issues: n/a
+- Entity Issue: [20937](https://github.com/bcgov/entity/issues/20937)
+- Implementation PR: (leave this empty)
+
+
+## Overview
+
+The involuntary dissolution process will consist of two BE scheduled jobs - involuntary dissolutions job and furnishings job.
+
+This process will be mostly automated with staff input only in the form of updating a configuration.  The configuration determines how many new businesses can be added to the dissolution process on a per day basis.
+
+The involuntary dissolutions job will manage the overall dissolution process for batches of businesses where the dissolution process is kicked off.
+
+The furnishings job will manage and track the notifications that need to go out as a part of the dissolution process.  This will involve emails, physical mail and any other required notifications made to external systems.  Functionality to reconcile BCMail+ physical mail outs and potentially track bounced emails(if required) may be handled in this job as well.
```

Comment:
Tracking bounced emails might be a different job, since those bounces will be async.

Also I don't know how you'd set up a mailbox to listen for bounces.

Reply:
Patrick apparently has something to track bounced emails.

I've only really brought this up to start surfacing questions that still need answers.  This will fit more in the design work I plan to do for the furnishing job.

But at the end of the day, getting an answer to if tracking bounced emails is a business requirement is the first priority.  If it's not required, all this just goes away.

Reply:
OK. Can you update to make it more clear that the first job might send emails but this second job might handle bounces?

Reply:
Updated furnishing job section to be more clear as it relates to emails.

The furnishing job will handle all the outbound notifications.  

I think if it we do check for bounced emails it will be in the furnishings job.  As it runs once a day, the bounce check can be done the next day for notifications that went out the previous day.